lib/modules/platform/github/index.ts

```diff
@@ -121,10 +121,18 @@ export async function initPlatform({
   gitAuthor,
 }: PlatformParams): Promise<PlatformResult> {
   if (!token) {
-    throw new Error('Init: You must configure a GitHub personal access token');
+    throw new Error(
+      'Init: You must configure a GitHub personal access token or app installation token'
```

Comment:
```suggestion
      'Init: You must configure a GitHub token'
```

You'll need to reformat plus update the snapshots

Reply:
I addressed it in the latest commit 900e90d